### ISSUE TITLE
Windows not extending partition

### ISSUE THREAD

**Issue by sgocken** [OpenNebula/addon-context-windows#45]
The PowerShell script would not extend partitions on any of my Windows 10 or Server 2016 templates.

I found the root cause and made a patch.

Cause a hidden reserved partition before the main OS partition

wmic does not show it:
```
wmic partition where DiskIndex=0 get Index,Caption

Caption                Index
Disk #0, Partition #0  0
Disk #0, Partition #1  1
Disk #0, Partition #2  2
```

Where diskpart does:
```
"select disk 0","list partition"| diskpart | Select-String -Pattern "^  Partition \d" -AllMatches

  Partition 1    Recovery           499 MB  1024 KB
  Partition 2    System              99 MB   500 MB
  Partition 3    Reserved            16 MB   599 MB
  Partition 4    Primary            197 GB   615 MB
```

My changed version of the function extendPartitions: 
I rewrote the $partsIds list collection using just diskpart.
```
function extendPartitions()
{
    Write-Output "- Extend partitions"

    #$diskIds = ((wmic diskdrive get Index | Select-String "[0-9]+") -replace '\D','')
    $diskId = 0

    #$partIds = ((wmic partition where DiskIndex=$diskId get Index | Select-String "[0-9]+") -replace '\D','' | %{[int]$_ + 1})
    $partIds = "select disk $diskId","list partition"| diskpart | Select-String -Pattern "^  Partition \d" -AllMatches | %{$_.matches} | %{$_.value.replace("  Partition ", "") }

    ForEach ($partId in $partIds) {
        extendPartition $diskId $partId
    }
}
```

**Comment by linjan** [comment]
Hello, does this patch included in 5.7.80 or 5.6.0?

**Comment by al3xhh** [comment]
Hello @linjan, the changes has been made and the bug is fixed, it will be available in the 5.8.0 context packages.

Thanks for your help!
